content/200-orm/200-prisma-client/500-deployment/301-edge/100-overview.mdx

```diff
@@ -0,0 +1,58 @@
+
+## Edge-compatibility of database drivers
+
+### Why are there limitations around database drivers in edge functions?
+
+Edge functions typically don't use the standard Node.js runtime. For example, Vercel Edge Functions and Cloudflare Workers are running code in [V8 isolates](https://v8docs.nodesource.com/node-0.8/d5/dda/classv8_1_1_isolate.html). Deno Deploy is using the [Deno](https://deno.com/) JavaScript runtime. As a consequence, these edge functions only have access to a small subset of the standard Node.js APIs and also have constrained computing resources (CPU and memory).
+
+In particular, the constraint of not being able to freely open TCP connections makes it difficult to talk to a traditional database from an edge function. While Cloudflare has introduced a [`connect()`](https://developers.cloudflare.com/workers/runtime-apis/tcp-sockets/) API that enables limited TCP connections, this still only enables database access using specific database drivers that are compatible with that API.
+
+> **Note**: [Prisma Accelerate](/accelerate) enables you to access _any_ database from _any_ edge function provider. No edge-compatible driver is necessary.
+
+### Which database drivers are edge-compatible?
+
+Here is an overview of the different database drivers and their compatibility with different edge function offerings:
+
+- [Neon Serverless](https://neon.tech/docs/serverless/serverless-driver) uses HTTP to access the database
+- [PlanetScale Serverless](https://planetscale.com/docs/tutorials/planetscale-serverless-driver) uses HTTP to access the database
+- [`node-postgres`](https://node-postgres.com/) (`pg`) uses Cloudflare's `connect()` (TCP) to access the database
+- [`@libsql/client`](https://github.com/tursodatabase/libsql-client-ts) is used to accessed Turso databases
+
+> **Note**: There's [also work being done](https://github.com/sidorares/node-mysql2/pull/2289) on the `node-mysql2` driver which will enable to access traditional MySQL databases from edge functions as well.
+
+You can use all of these drivers with Prisma ORM using the respective [driver adapters](/orm/overview/databases/database-drivers).
+
+Depending on which deployment provider and database/driver you use, there may be special considerations. Please take a look at the deployment docs for your respective scenario to make sure you can deploy your application successfully:
+
+- Cloudflare
+  - [PostgreSQL (traditional)](/orm/prisma-client/deployment/edge/deploy-to-cloudflare#postgresql-traditional)
+  - [PlanetScale](/orm/prisma-client/deployment/edge/deploy-to-cloudflare#planetscale)
+  - [Neon](/orm/prisma-client/deployment/edge/deploy-to-cloudflare#neon)
+- Vercel
+  - [Vercel Postgres](/orm/prisma-client/deployment/edge/deploy-to-vercel#vercel-postgres)
+  - [Neon](/orm/prisma-client/deployment/edge/deploy-to-vercel#neon)
+  - [PlanetScale](/orm/prisma-client/deployment/edge/deploy-to-vercel#planetscale)
```

Comment:
This listing should probably also include Turso for both?

Reply:
since this is a list that links to the respective docs, including Turso wouldn't make sense here IMO as long as we don't have any docs for it. 

that brings up the question: should we include Turso deployment instructions as well? I didn't have any information about Turso so far but if we need them, I can add the instructions for Turso to the docs as well. wdyt?

Reply:
We have general documentation for Turso: https://www.prisma.io/docs/orm/overview/databases/turso

As it is one of the supported databases on edge functions, I think we should also mention it here, yes.

Reply:
I've linked to these general docs for now and created an issue for the concrete instructions: https://github.com/prisma/docs/issues/5709